pkgs/tools/misc/fzf/default.nix

```diff
@@ -64,6 +66,7 @@ buildGoModule rec {
 
   postInstall = ''
     install bin/fzf-tmux $out/bin
+    wrapProgram $out/bin/fzf-tmux --prefix PATH : "${lib.makeBinPath [ bc ]}"
```

Comment:
bc is just used here https://github.com/junegunn/fzf/blob/master/bin/fzf-tmux#L183 so patching would probably be a better idea

Reply:
Replaced it with a `substituteInPlace`

```diff
--- /home/msfjarvis/.nix-profile/bin/fzf-tmux	1970-01-01 05:30:01.000000000 +0530
+++ ./result/bin/fzf-tmux	1970-01-01 05:30:01.000000000 +0530
@@ -180,7 +180,7 @@
 envs="export TERM=$TERM "
 if [[ "$opt" =~ "-E" ]]; then
   tmux_version=$(tmux -V | sed 's/[^0-9.]//g')
-  if [[ $(awk '{print ($1 > 3.2)}' <<< "$tmux_version" 2> /dev/null || bc -l <<< "$tmux_version > 3.2") = 1 ]]; then
+  if [[ $(awk '{print ($1 > 3.2)}' <<< "$tmux_version" 2> /dev/null || /nix/store/fqba9jy3d6f2p67qhdmbdfmvpmrvyx1x-bc-1.07.1/bin/bc -l <<< "$tmux_version > 3.2") = 1 ]]; then
     FZF_DEFAULT_OPTS="--border $FZF_DEFAULT_OPTS"
     opt="-B $opt"
   elif [[ $tmux_version = 3.2 ]]; then
```